index.js

```diff
@@ -38,6 +38,19 @@ var argv = require('yargs')
     default: false,
     global: true
   })
+  .option('pre-release', {
+    alias: 'p',
+    describe: 'Should the release be a pre-release?',
+    type: 'boolean',
+    default: false,
+    global: true
+  })
+  .option('pre-id', {
```

Comment:
Where did you see the name "pre-id"?



> About the publish, yes but this is in the users scope.

What I mean is just to display a recommendation. Because right now it says [`npm publish` without a tag](https://github.com/conventional-changelog/standard-version/blob/master/index.js#L158-L160). Displaying `--tag` would be a better recommendation.